src/js/components/bulkDownload/awards/filters/AwardLevelAndTypeFilter.jsx

```diff
@@ -0,0 +1,81 @@
+/**
+ * AwardLevelAndTypeFilter.jsx
+ * Created by Seth Stoudenmier 03/01/20
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { CheckCircle, ExclamationCircle } from 'components/sharedComponents/icons/Icons';
+import PrimaryCheckboxType from '../../../sharedComponents/checkbox/PrimaryCheckboxType';
+
+const propTypes = {
+    awardLevels: PropTypes.array,
+    awardTypeLabels: PropTypes.object,
+    currentAwardTypes: PropTypes.object,
+    bulkPrimeAwardTypeChange: PropTypes.func,
+    bulkSubAwardTypeChange: PropTypes.func,
+    togglePrimeAwardTypeChange: PropTypes.func,
+    toggleSubAwardTypeChange: PropTypes.func
+};
+
+export default class AwardLevelAndTypeFilter extends React.Component {
```

Comment:
Not a change request, but we would typically make this a stateless functional component. This is where the react ecosystem is going.